contrib/mastoarch

```diff
@@ -8,20 +8,33 @@
 # -----------------------------------------------------------------------------
 # Default options (can be overridden via command-line):
 myacc="Demo@server.tld" # Your (default) Mastodon handle
+archive_followers=0     # whether to archive followers as well
 autogit=0               # whether to automatically check-in new data to your local git repo
 
 # !!! ================================================================== !!!
 # !!! Do not change below this line unless you know what you're doing :) !!!
 # !!! ================================================================== !!!
 
+# -----------------------------------------------------------------------------
+# Load config if exists
+[[ -f "${HOME}/.config/mastodon-backup/config" ]] && source "${HOME}/.config/mastodon-backup/config"
```

Comment:
I know that the path on GitHub still says mastodon-backup but everywhere else the name has been changed to mastodon-archive.

Reply:
So shall I change all paths accordingly? `${HOME}/.config/mastodon-archive` as config dir, for consistency? No big deal at this point (as I'm most likely the only one using it currently).